### PR TITLE
Fixed first example in readme

### PR DESCRIPTION
**Pull request by AmyrAhmady**
a fixed and usable one, without any needs when you want to test it.
previous PR closed due to being incomplete.

### CODE REVIEW

**Comment by Jacse** [comment]
The main function of the first example is to very quickly allow people to understand what the component is and what it does. I think more code takes away that focus.

On the other hand, as you mention, the current example is not runnable. What I propose we do, as has been done in later examples, is to instead add a line a la ``// const styles = ...``. That makes it clear that styling needs to be setup without adding extra code

File: README.md [hunk 1]
```diff
@@ -59,8 +79,13 @@ const slides = [
 ];
 
 export default class App extends React.Component {
-  this.state = {
-    showRealApp: false
+
```
Comment (Jacse) [inline README.md]: Adding a constructor function is actually not necessary for initializing state, so this change can be removed.

**Comment by Jacse** [comment]
I'm going to close this since it doesn't create much additional value and there has been no activity. Let me know if you still want to go ahead.

**Comment by philipp65** [comment]
Actually I think @AmyrAhmady is right by changing the first example. It looks like the complete class but misses imports and styles.

So you either have to complete the example or comment missing stuff or use `...` as indication of missing code

**Pull request closed without merging**